cachi2/core/package_managers/yarn/main.py

```diff
@@ -13,7 +13,11 @@
     get_semver_from_yarn_path,
 )
 from cachi2.core.package_managers.yarn.resolver import create_components, resolve_packages
-from cachi2.core.package_managers.yarn.utils import run_yarn_cmd
+from cachi2.core.package_managers.yarn.utils import (
```

Comment:
This has an identical commit subject as the previous commit, was that intentional? It may look weird in the condensed git history dump so one of them should be tweaked ever so slightly.

Reply:
No, it was not. Fixed.

Reply:
is it ? or I am blind

Reply:
Not yet, I am still re-shaping the two commits, but it will be fixed in the next push

Reply:
Now should be done.